userspace/libsinsp/sinsp.cpp

```diff
@@ -646,6 +649,8 @@ void sinsp::open_nodriver()
 		oargs.proc_callback_context = this;
 	}
 	oargs.import_users = m_usergroup_manager.m_import_users;
+	oargs.ppm_sc_of_interest = NULL;
```

Comment:
Not sure what is the best strategy here, default the syscalls in sinsp or pass a NULL pointer to scap that will manage it 🤔 Probably defaulting them here could avoid us some checks in different engines, we could do it here in just one place... any ideas on that?

Reply:
Basically, i needed this for the `tp_of_interest` because IMO sinsp should not care about it.
Then i realized that it was best to do the same for `ppm_sc_of_interest` too, ie: zeroed value (NULL) means default behavior that is catch all.

Reply:
Right now we cannot modify `tp_of_interest` at run-time you can only configure the initial set when you open the specific engine `open_bpf(...,tp_set,...)`, while for the interesting syscall we will have a new API 

```c
void sinsp::mark_ppm_sc_of_interest(uint32_t ppm_sc, bool enable)
```

that allows you to disable and enable the single syscall during the execution with the `ppm_code`